github/orgs_projects.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // ListProjectsOptions specifies the optional parameters to the
 // OrganizationsService.ListProjects method.
-type ListProjectsOptions struct {
+type ProjectsListOptions struct {
```

Comment:
> Are we ok leaving the definition in orgs_projects.go?

What I mean by this is that `ProjectListOptions` is used by `orgs_projects.go` and `repo_projects.go`, but is defined in `orgs_projects.go`... just wanted to be sure we didn't want to move it to somewhere else, since it's shared between the files.